### ISSUE TITLE
Native apps on Win10 crash using 1.0.6+

### ISSUE THREAD

**Issue by jan-tosovsky-cz** [gluonhq/gluonfx-maven-plugin#380]
This seems to be a regression as I could run my JavaFX native app built using 1.0.5.

This works:
javafx-controls: 17.0.1
javafx-fxml: 17.0.1
javafx-maven-plugin: 0.0.7
gluonfx-maven-plugin: 1.0.5

This fails:
javafx-controls: 17.0.1
javafx-fxml: 17.0.1
javafx-maven-plugin: 0.0.7
gluonfx-maven-plugin: 1.0.6+

In WinDbg tool I can see the progress of loading libraries, but the last line before crash is different per project:
* `dxcore.dll` (private project, but I can share it to whoever is willing to debug it, the complete trace is below)
* `rtscom.dll` (public project, https://github.com/drifted-in/jet-browser-javafx )

```
Microsoft (R) Windows Debugger Version 10.0.22415.1003 AMD64
Copyright (c) Microsoft Corporation. All rights reserved.

CommandLine: C:\tools\tools-job-ticket-maker\target\gluonfx\x86_64-windows\job-ticket-maker.exe

************* Path validation summary **************
Response                         Time (ms)     Location
Deferred                                       srv*
Symbol search path is: srv*
Executable search path is: 
ModLoad: 00007ff6`01d20000 00007ff6`05d2a000   job-ticket-maker.exe
ModLoad: 00007ffb`35630000 00007ffb`35825000   ntdll.dll
ModLoad: 00007ffb`353e0000 00007ffb`3549e000   C:\Windows\System32\KERNEL32.DLL
ModLoad: 00007ffb`32d50000 00007ffb`33019000   C:\Windows\System32\KERNELBASE.dll
ModLoad: 00007ffb`35540000 00007ffb`355ec000   C:\Windows\System32\ADVAPI32.dll
ModLoad: 00007ffb`34a00000 00007ffb`34a9e000   C:\Windows\System32\msvcrt.dll
ModLoad: 00007ffb`354a0000 00007ffb`3553b000   C:\Windows\System32\sechost.dll
ModLoad: 00007ffb`33fb0000 00007ffb`340da000   C:\Windows\System32\RPCRT4.dll
ModLoad: 00007ffb`34990000 00007ffb`349fb000   C:\Windows\System32\WS2_32.dll
ModLoad: 00007ffb`34730000 00007ffb`3480a000   C:\Windows\System32\COMDLG32.dll
ModLoad: 00007ffb`2c710000 00007ffb`2c71a000   C:\Windows\SYSTEM32\VERSION.dll
ModLoad: 00007ffb`32c50000 00007ffb`32c7e000   C:\Windows\SYSTEM32\USERENV.dll
ModLoad: 00007ffb`34340000 00007ffb`34695000   C:\Windows\System32\combase.dll
ModLoad: 00007ffb`33320000 00007ffb`33420000   C:\Windows\System32\ucrtbase.dll
ModLoad: 00007ffb`340e0000 00007ffb`3418e000   C:\Windows\System32\shcore.dll
ModLoad: 00007ffb`34190000 00007ffb`34331000   C:\Windows\System32\USER32.dll
ModLoad: 00007ffb`33080000 00007ffb`330a2000   C:\Windows\System32\win32u.dll
ModLoad: 00007ffb`34ba0000 00007ffb`34bcb000   C:\Windows\System32\GDI32.dll
ModLoad: 00007ffb`33210000 00007ffb`3331b000   C:\Windows\System32\gdi32full.dll
ModLoad: 00007ffb`33550000 00007ffb`335ed000   C:\Windows\System32\msvcp_win.dll
ModLoad: 00007ffb`35380000 00007ffb`353d5000   C:\Windows\System32\SHLWAPI.dll
ModLoad: 00007ffb`337f0000 00007ffb`33f2f000   C:\Windows\System32\SHELL32.dll
ModLoad: 00007ffa`f8110000 00007ffa`f81c0000   C:\Windows\WinSxS\amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.19041.1110_none_792d1c772443f647\COMCTL32.dll
ModLoad: 00007ffb`346a0000 00007ffb`346d0000   C:\Windows\System32\IMM32.dll
ModLoad: 00007ffb`33690000 00007ffb`337ba000   C:\Windows\System32\ole32.dll
ModLoad: 00007ffb`30a70000 00007ffb`30a9f000   C:\Windows\SYSTEM32\dwmapi.dll
ModLoad: 00007ffb`35200000 00007ffb`352cd000   C:\Windows\System32\OLEAUT32.dll
ModLoad: 00007ffb`21040000 00007ffb`2122b000   C:\Windows\SYSTEM32\urlmon.dll
ModLoad: 00007ffb`04490000 00007ffb`04786000   C:\Windows\SYSTEM32\UIAutomationCore.DLL
ModLoad: 00007ffb`12f20000 00007ffb`12f47000   C:\Windows\SYSTEM32\WINMM.dll
ModLoad: 00007ffb`26760000 00007ffb`267ed000   C:\Windows\SYSTEM32\MSVCP140.dll
ModLoad: 00007ffb`267f0000 00007ffb`2680b000   C:\Windows\SYSTEM32\VCRUNTIME140.dll
ModLoad: 00007ffb`266e0000 00007ffb`266ec000   C:\Windows\SYSTEM32\VCRUNTIME140_1.dll
ModLoad: 00007ffb`32430000 00007ffb`3249a000   C:\Windows\SYSTEM32\MSWSOCK.dll
ModLoad: 00007ffb`32120000 00007ffb`3215b000   C:\Windows\SYSTEM32\IPHLPAPI.DLL
ModLoad: 00007ffb`2e080000 00007ffb`2e176000   C:\Windows\SYSTEM32\PROPSYS.dll
ModLoad: 00007ffb`22e10000 00007ffb`230c0000   C:\Windows\SYSTEM32\iertutil.dll
ModLoad: 00007ffb`23840000 00007ffb`2384c000   C:\Windows\SYSTEM32\Secur32.dll
ModLoad: 00007ffb`32230000 00007ffb`3223c000   C:\Windows\SYSTEM32\netutils.dll
ModLoad: 00007ffb`265d0000 00007ffb`266d8000   C:\Windows\SYSTEM32\WINHTTP.dll
ModLoad: 00007ffb`24720000 00007ffb`24748000   C:\Windows\SYSTEM32\srvcli.dll
ModLoad: 00007ffb`32c10000 00007ffb`32c41000   C:\Windows\SYSTEM32\SSPICLI.DLL
(18b0.6e90): Break instruction exception - code 80000003 (first chance)
ntdll!LdrpDoDebuggerBreak+0x30:
00007ffb`357006b0 cc              int     3
0:000> g
ModLoad: 00007ffb`30ca0000 00007ffb`30cb2000   C:\Windows\SYSTEM32\kernel.appcore.dll
ModLoad: 00007ffb`32c90000 00007ffb`32caf000   C:\Windows\SYSTEM32\profapi.dll
ModLoad: 000001c0`aedf0000 000001c0`aedf3000   C:\Program Files\PowerShell\7\api-ms-win-core-console-l1-1-0.dll
ModLoad: 000001c0`aedf0000 000001c0`aedf3000   C:\Program Files\PowerShell\7\api-ms-win-core-console-l1-1-0.dll
ModLoad: 000001c0`aee00000 000001c0`aee03000   C:\Program Files\PowerShell\7\api-ms-win-core-console-l1-2-0.dll
ModLoad: 000001c0`aee00000 000001c0`aee03000   C:\Program Files\PowerShell\7\api-ms-win-core-console-l1-2-0.dll
ModLoad: 000001c0`aee10000 000001c0`aee13000   C:\Program Files\PowerShell\7\api-ms-win-core-datetime-l1-1-0.dll
ModLoad: 000001c0`aee10000 000001c0`aee13000   C:\Program Files\PowerShell\7\api-ms-win-core-datetime-l1-1-0.dll
ModLoad: 000001c0`aee20000 000001c0`aee23000   C:\Program Files\PowerShell\7\api-ms-win-core-debug-l1-1-0.dll
ModLoad: 000001c0`aee20000 000001c0`aee23000   C:\Program Files\PowerShell\7\api-ms-win-core-debug-l1-1-0.dll
ModLoad: 000001c0`aee30000 000001c0`aee33000   C:\Program Files\PowerShell\7\api-ms-win-core-errorhandling-l1-1-0.dll
ModLoad: 000001c0`aee30000 000001c0`aee33000   C:\Program Files\PowerShell\7\api-ms-win-core-errorhandling-l1-1-0.dll
ModLoad: 000001c0`aee40000 000001c0`aee44000   C:\Program Files\PowerShell\7\api-ms-win-core-file-l1-1-0.dll
ModLoad: 000001c0`aee40000 000001c0`aee44000   C:\Program Files\PowerShell\7\api-ms-win-core-file-l1-1-0.dll
ModLoad: 000001c0`aee50000 000001c0`aee53000   C:\Program Files\PowerShell\7\api-ms-win-core-file-l1-2-0.dll
ModLoad: 000001c0`aee50000 000001c0`aee53000   C:\Program Files\PowerShell\7\api-ms-win-core-file-l1-2-0.dll
ModLoad: 000001c0`aee60000 000001c0`aee63000   C:\Program Files\PowerShell\7\api-ms-win-core-file-l2-1-0.dll
ModLoad: 000001c0`aee60000 000001c0`aee63000   C:\Program Files\PowerShell\7\api-ms-win-core-file-l2-1-0.dll
ModLoad: 000001c0`af070000 000001c0`af073000   C:\Program Files\PowerShell\7\api-ms-win-core-handle-l1-1-0.dll
ModLoad: 000001c0`af070000 000001c0`af073000   C:\Program Files\PowerShell\7\api-ms-win-core-handle-l1-1-0.dll
ModLoad: 000001c0`af080000 000001c0`af083000   C:\Program Files\PowerShell\7\api-ms-win-core-heap-l1-1-0.dll
ModLoad: 000001c0`af080000 000001c0`af083000   C:\Program Files\PowerShell\7\api-ms-win-core-heap-l1-1-0.dll
ModLoad: 000001c0`af090000 000001c0`af093000   C:\Program Files\PowerShell\7\api-ms-win-core-interlocked-l1-1-0.dll
ModLoad: 000001c0`af090000 000001c0`af093000   C:\Program Files\PowerShell\7\api-ms-win-core-interlocked-l1-1-0.dll
ModLoad: 000001c0`af0a0000 000001c0`af0a3000   C:\Program Files\PowerShell\7\api-ms-win-core-libraryloader-l1-1-0.dll
ModLoad: 000001c0`af0a0000 000001c0`af0a3000   C:\Program Files\PowerShell\7\api-ms-win-core-libraryloader-l1-1-0.dll
ModLoad: 000001c0`b3790000 000001c0`b3793000   C:\Program Files\PowerShell\7\api-ms-win-core-localization-l1-2-0.dll
ModLoad: 000001c0`b3790000 000001c0`b3793000   C:\Program Files\PowerShell\7\api-ms-win-core-localization-l1-2-0.dll
ModLoad: 000001c0`b37a0000 000001c0`b37a3000   C:\Program Files\PowerShell\7\api-ms-win-core-memory-l1-1-0.dll
ModLoad: 000001c0`b37a0000 000001c0`b37a3000   C:\Program Files\PowerShell\7\api-ms-win-core-memory-l1-1-0.dll
ModLoad: 000001c0`b37b0000 000001c0`b37b3000   C:\Program Files\PowerShell\7\api-ms-win-core-namedpipe-l1-1-0.dll
ModLoad: 000001c0`b37b0000 000001c0`b37b3000   C:\Program Files\PowerShell\7\api-ms-win-core-namedpipe-l1-1-0.dll
ModLoad: 000001c0`b39c0000 000001c0`b39c3000   C:\Program Files\PowerShell\7\api-ms-win-core-processenvironment-l1-1-0.dll
ModLoad: 000001c0`b39c0000 000001c0`b39c3000   C:\Program Files\PowerShell\7\api-ms-win-core-processenvironment-l1-1-0.dll
ModLoad: 000001c0`b39d0000 000001c0`b39d3000   C:\Program Files\PowerShell\7\api-ms-win-core-processthreads-l1-1-0.dll
ModLoad: 000001c0`b39d0000 000001c0`b39d3000   C:\Program Files\PowerShell\7\api-ms-win-core-processthreads-l1-1-0.dll
ModLoad: 000001c0`b39e0000 000001c0`b39e3000   C:\Program Files\PowerShell\7\api-ms-win-core-processthreads-l1-1-1.dll
ModLoad: 000001c0`b39e0000 000001c0`b39e3000   C:\Program Files\PowerShell\7\api-ms-win-core-processthreads-l1-1-1.dll
ModLoad: 000001c0`b39f0000 000001c0`b39f3000   C:\Program Files\PowerShell\7\api-ms-win-core-profile-l1-1-0.dll
ModLoad: 000001c0`b39f0000 000001c0`b39f3000   C:\Program Files\PowerShell\7\api-ms-win-core-profile-l1-1-0.dll
ModLoad: 000001c0`b3c00000 000001c0`b3c03000   C:\Program Files\PowerShell\7\api-ms-win-core-rtlsupport-l1-1-0.dll
ModLoad: 000001c0`b3c00000 000001c0`b3c03000   C:\Program Files\PowerShell\7\api-ms-win-core-rtlsupport-l1-1-0.dll
ModLoad: 000001c0`b3c10000 000001c0`b3c13000   C:\Program Files\PowerShell\7\api-ms-win-core-string-l1-1-0.dll
ModLoad: 000001c0`b3c10000 000001c0`b3c13000   C:\Program Files\PowerShell\7\api-ms-win-core-string-l1-1-0.dll
ModLoad: 000001c0`b3c20000 000001c0`b3c23000   C:\Program Files\PowerShell\7\api-ms-win-core-synch-l1-1-0.dll
ModLoad: 000001c0`b3c20000 000001c0`b3c23000   C:\Program Files\PowerShell\7\api-ms-win-core-synch-l1-1-0.dll
ModLoad: 000001c0`b3c30000 000001c0`b3c33000   C:\Program Files\PowerShell\7\api-ms-win-core-synch-l1-2-0.dll
ModLoad: 000001c0`b3c30000 000001c0`b3c33000   C:\Program Files\PowerShell\7\api-ms-win-core-synch-l1-2-0.dll
ModLoad: 000001c0`b3e40000 000001c0`b3e43000   C:\Program Files\PowerShell\7\api-ms-win-core-sysinfo-l1-1-0.dll
ModLoad: 000001c0`b3e40000 000001c0`b3e43000   C:\Program Files\PowerShell\7\api-ms-win-core-sysinfo-l1-1-0.dll
ModLoad: 000001c0`b3e50000 000001c0`b3e53000   C:\Program Files\PowerShell\7\api-ms-win-core-timezone-l1-1-0.dll
ModLoad: 000001c0`b3e50000 000001c0`b3e53000   C:\Program Files\PowerShell\7\api-ms-win-core-timezone-l1-1-0.dll
ModLoad: 000001c0`b3e60000 000001c0`b3e63000   C:\Program Files\PowerShell\7\api-ms-win-core-util-l1-1-0.dll
ModLoad: 000001c0`b3e60000 000001c0`b3e63000   C:\Program Files\PowerShell\7\api-ms-win-core-util-l1-1-0.dll
ModLoad: 000001c0`b3e70000 000001c0`b3e73000   C:\Program Files\PowerShell\7\api-ms-win-crt-conio-l1-1-0.dll
ModLoad: 000001c0`b3e70000 000001c0`b3e73000   C:\Program Files\PowerShell\7\api-ms-win-crt-conio-l1-1-0.dll
ModLoad: 000001c0`b4080000 000001c0`b4084000   C:\Program Files\PowerShell\7\api-ms-win-crt-convert-l1-1-0.dll
ModLoad: 000001c0`b4080000 000001c0`b4084000   C:\Program Files\PowerShell\7\api-ms-win-crt-convert-l1-1-0.dll
ModLoad: 000001c0`b4090000 000001c0`b4093000   C:\Program Files\PowerShell\7\api-ms-win-crt-environment-l1-1-0.dll
ModLoad: 000001c0`b4090000 000001c0`b4093000   C:\Program Files\PowerShell\7\api-ms-win-crt-environment-l1-1-0.dll
ModLoad: 000001c0`b40a0000 000001c0`b40a3000   C:\Program Files\PowerShell\7\api-ms-win-crt-filesystem-l1-1-0.dll
ModLoad: 000001c0`b40a0000 000001c0`b40a3000   C:\Program Files\PowerShell\7\api-ms-win-crt-filesystem-l1-1-0.dll
ModLoad: 000001c0`b40b0000 000001c0`b40b3000   C:\Program Files\PowerShell\7\api-ms-win-crt-heap-l1-1-0.dll
ModLoad: 000001c0`b40b0000 000001c0`b40b3000   C:\Program Files\PowerShell\7\api-ms-win-crt-heap-l1-1-0.dll
ModLoad: 000001c0`b42c0000 000001c0`b42c3000   C:\Program Files\PowerShell\7\api-ms-win-crt-locale-l1-1-0.dll
ModLoad: 000001c0`b42c0000 000001c0`b42c3000   C:\Program Files\PowerShell\7\api-ms-win-crt-locale-l1-1-0.dll
ModLoad: 000001c0`b42d0000 000001c0`b42d5000   C:\Program Files\PowerShell\7\api-ms-win-crt-math-l1-1-0.dll
ModLoad: 000001c0`b42d0000 000001c0`b42d5000   C:\Program Files\PowerShell\7\api-ms-win-crt-math-l1-1-0.dll
ModLoad: 000001c0`b42e0000 000001c0`b42e5000   C:\Program Files\PowerShell\7\api-ms-win-crt-multibyte-l1-1-0.dll
ModLoad: 000001c0`b42e0000 000001c0`b42e5000   C:\Program Files\PowerShell\7\api-ms-win-crt-multibyte-l1-1-0.dll
ModLoad: 000001c0`b42f0000 000001c0`b4300000   C:\Program Files\PowerShell\7\api-ms-win-crt-private-l1-1-0.dll
ModLoad: 000001c0`b42f0000 000001c0`b4300000   C:\Program Files\PowerShell\7\api-ms-win-crt-private-l1-1-0.dll
ModLoad: 000001c0`b4300000 000001c0`b4303000   C:\Program Files\PowerShell\7\api-ms-win-crt-process-l1-1-0.dll
ModLoad: 000001c0`b4300000 000001c0`b4303000   C:\Program Files\PowerShell\7\api-ms-win-crt-process-l1-1-0.dll
ModLoad: 000001c0`b4510000 000001c0`b4514000   C:\Program Files\PowerShell\7\api-ms-win-crt-runtime-l1-1-0.dll
ModLoad: 000001c0`b4510000 000001c0`b4514000   C:\Program Files\PowerShell\7\api-ms-win-crt-runtime-l1-1-0.dll
ModLoad: 000001c0`b4520000 000001c0`b4524000   C:\Program Files\PowerShell\7\api-ms-win-crt-stdio-l1-1-0.dll
ModLoad: 000001c0`b4520000 000001c0`b4524000   C:\Program Files\PowerShell\7\api-ms-win-crt-stdio-l1-1-0.dll
ModLoad: 000001c0`b4530000 000001c0`b4534000   C:\Program Files\PowerShell\7\api-ms-win-crt-string-l1-1-0.dll
ModLoad: 000001c0`b4530000 000001c0`b4534000   C:\Program Files\PowerShell\7\api-ms-win-crt-string-l1-1-0.dll
ModLoad: 000001c0`b4540000 000001c0`b4543000   C:\Program Files\PowerShell\7\api-ms-win-crt-time-l1-1-0.dll
ModLoad: 000001c0`b4540000 000001c0`b4543000   C:\Program Files\PowerShell\7\api-ms-win-crt-time-l1-1-0.dll
ModLoad: 000001c0`b4750000 000001c0`b4753000   C:\Program Files\PowerShell\7\api-ms-win-crt-utility-l1-1-0.dll
ModLoad: 000001c0`b4750000 000001c0`b4753000   C:\Program Files\PowerShell\7\api-ms-win-crt-utility-l1-1-0.dll
ModLoad: 00007ffa`eff40000 00007ffa`f010e000   C:\Windows\system32\d3d9.dll
ModLoad: 00007ffb`30e50000 00007ffb`315e0000   C:\Windows\SYSTEM32\windows.storage.dll
ModLoad: 00007ffb`326d0000 00007ffb`326fc000   C:\Windows\SYSTEM32\Wldp.dll
ModLoad: 00007ffb`306c0000 00007ffb`3075e000   C:\Windows\system32\uxtheme.dll
ModLoad: 00007ffa`b6a10000 00007ffa`b6b73000   C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a086f01cc7be643a\igdumdim64.dll
ModLoad: 00007ffa`9afd0000 00007ffa`9e9d4000   C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a086f01cc7be643a\igd9dxva64.dll
ModLoad: 00007ffb`34d80000 00007ffb`351f2000   C:\Windows\System32\SETUPAPI.dll
ModLoad: 00007ffb`33420000 00007ffb`3346e000   C:\Windows\System32\cfgmgr32.dll
ModLoad: 00007ffb`33470000 00007ffb`33497000   C:\Windows\System32\bcrypt.dll
ModLoad: 00007ffa`ff1a0000 00007ffa`ff3df000   C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a086f01cc7be643a\igdgmm64.dll
ModLoad: 00007ffb`34810000 00007ffb`34925000   C:\Windows\System32\MSCTF.dll
ModLoad: 00007ffb`335f0000 00007ffb`33673000   C:\Windows\System32\bcryptPrimitives.dll
ModLoad: 00007ffb`30790000 00007ffb`307a4000   C:\Windows\SYSTEM32\resourcepolicyclient.dll
ModLoad: 00007ffb`06390000 00007ffb`063b7000   C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a086f01cc7be643a\igdinfo64.dll
ModLoad: 00007ffa`fbd00000 00007ffa`fe962000   C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_a086f01cc7be643a\igc64.dll
ModLoad: 00007ffb`2c660000 00007ffb`2c69b000   C:\Windows\SYSTEM32\dxcore.dll
ntdll!NtTerminateProcess+0x14:
00007ffb`356cd2f4 c3              ret
```
There is also some stracktrace in WinDbg:
```
[0x0]   ntdll!NtTerminateProcess + 0x14   
[0x1]   ntdll!RtlExitUserProcess + 0xb8   
[0x2]   KERNEL32!ExitProcessImplementation + 0xb   
[0x3]   ucrtbase!exit_or_terminate_process + 0x44   
[0x4]   ucrtbase!common_exit + 0x6f   
[0x5]   job_ticket_maker!JVM_Yield + 0xe8b   
[0x6]   KERNEL32!BaseThreadInitThunk + 0x14   
[0x7]   ntdll!RtlUserThreadStart + 0x21  
 ```

**Comment by jan-tosovsky-cz** [comment]
Thanks, the issue seems to be fixed using the current 1.0.10 on a laptop where I built that EXE.
But when running the same EXE on a different laptop (also Win 10 64-bit), the window is not shown. In the error log there isn't anything special:
```
Dec 13, 2021 9:42:30 PM com.sun.javafx.application.PlatformImpl startup
WARNING: Unsupported JavaFX configuration: classes were loaded from 'unnamed module @5bdd5689'
``` 
In WinDbg I can see the list of loaded modules very reduced, the last one before the fail is: 
```
ModLoad: 00007ff9`33bd0000 00007ff9`33be2000   C:\Windows\SYSTEM32\kernel.appcore.dll
```
In the first laptop this list continues with
``` 
ModLoad: ....   C:\Windows\SYSTEM32\profapi.dll
```
(but the load order on both laptops is not identical, so not sure this is relevant here)

**Comment by Hugolarson** [comment]
Hi,

I still have problem with this under windows 10.
The .exe just wont start. 

s÷n juni 12 21:20:46 ICT 2022][INFO] ==================== RUN TASK ====================
[s÷n juni 12 21:20:46 ICT 2022][INFO] [SUB] Jun 12, 2022 9:20:46 PM com.sun.javafx.application.PlatformImpl startup
[s÷n juni 12 21:20:46 ICT 2022][INFO] [SUB] WARNING: Unsupported JavaFX configuration: classes were loaded from 'unnamed module @129a8472'
[s÷n juni 12 21:20:48 ICT 2022][SEVERE] Process run until end failed with result: -1073741819
Check the log files under C:\temp\gluon-samples-master\HelloFX\target\gluonfx\x86_64-windows\gvm\log
And please check https://docs.gluonhq.com/ for more information.
[s÷n juni 12 21:20:48 ICT 2022][INFO] Logging process [run until end] to file: C:\temp\gluon-samples-master\HelloFX\target\gluonfx\log\process-run until end-1655043648715.log
